fireapp/lib/domain/models/qualification/qualification.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:flutter/foundation.dart';
+
+part 'qualification.freezed.dart';
+part 'qualification.g.dart';
+
+@freezed
+class Qualification with _$Qualification {
+  const factory Qualification({
+    required int id,
+    required String name,
+    required bool deleted,
+    @JsonKey(name: 'update_datetime') DateTime? updateDateTime,
```

Comment:
Is update_datetime and insert_datetime the term agreed with the backend team? Just confused since it was different in the previous PR.

This looks good otherwise!

Reply:
I have check with the backend code, I just use the name of it, the code is:
    update_date_time = Column(DateTime, name='last_update_datetime', default=datetime.now(), nullable=False)
    insert_date_time = Column(DateTime, name='created_datetime', default=datetime.now(), nullable=False)
But I am not sure who write this code so didn't check with them

Reply:
In that case, your fields should be named update_date_time and insert_date_time. Once you've pushed that change I'll approve and merge :)